jenkins/plugins.txt

```diff
@@ -1,2 +1,5 @@
-# If we need to monitor memory usage
-monitoring:1.67.0
+credentials-binding:1.13
```

Comment:
Should we add a comment explaining why we need this?  i.e. point to the Jira issue... and maybe briefly mention why the others are here too?